### PR TITLE
Developmentui

### PR DESCRIPTION
**Pull request by GAUTAMRAJU15**
the merge for development . pls review

### CODE REVIEW

File: src/components/Notification/Notification.scss [hunk 1]
```diff
@@ -11,7 +11,7 @@
     background-color: white !important;
     width: 35px !important;
     .react-flexible-switch-circle{
-      background-color: #097fff !important;
+      background-color: !important;
```
Comment (ShankyRana) [inline src/components/Notification/Notification.scss]: remove this.

File: src/components/Sidebar/Sidebar.js [hunk 2]
```diff
@@ -9,7 +10,10 @@ class Sidebar extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      width: window.innerWidth
+      width: window.innerWidth,
+      uniqueVisitors:20000,
```
Comment (ShankyRana) [inline src/components/Sidebar/Sidebar.js]: please use dynamic, static only for testing

File: src/components/Sidebar/Sidebar.js [hunk 3]
```diff
@@ -18,7 +22,7 @@ class Sidebar extends Component {
   }
 
   render() {
-    const { disableButton } = this.props;
+    var { disableButton, quotaPercentage=Math.round(100*this.state.uniqueVisitors/this.state.uniqueVisitorQouta) } = this.props;
```
Comment (ShankyRana) [inline src/components/Sidebar/Sidebar.js]: this is wrong, correct it.

File: src/components/Dashboard/Dashboard.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { browserHistory } from 'react-router';
-import { Row, Col } from 'react-bootstrap';
+import { Row, Col} from 'react-bootstrap';
```
Comment (ShankyRana) [inline src/components/Dashboard/Dashboard.js]: please mantain spacing and all.

File: src/components/Sidebar/Sidebar.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router';
 import { Influence } from 'img';
+import {Col, ProgressBar } from 'react-bootstrap';
```
Comment (ShankyRana) [inline src/components/Sidebar/Sidebar.js]: same spacing.

File: src/components/Campaign/Campaign.js [hunk 6]
```diff
@@ -11,7 +11,7 @@ import {
 import CardHeader from 'components/Template/card-with-header';
 import Button from 'components/Template/customButton';
 
-import { Integrations1 } from 'img';
+//import { Integrations1 } from 'img';
```
Comment (ShankyRana) [inline src/components/Campaign/Campaign.js]: remove comments

File: src/components/CapturePage/CapturePage.js [hunk 7]
```diff
@@ -143,7 +143,7 @@ class CapturePage extends Component{
   render(){
     const { error, lead } = this.state;
     return (
-      <div className="content">
+      <div className="">
```
Comment (ShankyRana) [inline src/components/CapturePage/CapturePage.js]: remove classname

File: src/components/DisplayPage/DisplayPage.js [hunk 8]
```diff
@@ -161,7 +161,7 @@ class DisplayPage extends Component{
   render(){
     const { error, displayUrl } = this.state;
     return (
-      <div className="content display-page">
+      <div className="">
```
Comment (ShankyRana) [inline src/components/DisplayPage/DisplayPage.js]: remove classname

File: src/components/NotificationSettings/NotificationConfigure/NotificationConfigure.scss [hunk 9]
```diff
@@ -24,10 +21,16 @@
...
 
+
+
+
+
 FPqR3dGiqJeA3dGi7MM9_0 .FPqR2B_4qJeA2B_47MM9_0 {
   pointer-events: auto;
   overflow: hidden;
   position: relative;
+
```
Comment (ShankyRana) [inline src/components/NotificationSettings/NotificationConfigure/NotificationConfigure.scss]: remove unnecessary spacing

File: src/components/Campaign/Campaign.js (hunk 6 above)
Comment (GAUTAMRAJU15) [reply src/components/Campaign/Campaign.js]: done

File: src/components/NotificationSettings/NotificationConfigure/NotificationDesign.js [hunk 10]
```diff
@@ -2,9 +2,12 @@ import React from 'react';
 
 const NotificationDesign = ({
   contentText,
-  visitorText,
+  visitorText = ' ' +visitorText+ ' ',
```
Comment (ShankyRana) [inline src/components/NotificationSettings/NotificationConfigure/NotificationDesign.js]: add spacing inside the html, not like this

File: src/components/Sidebar/Sidebar.js (hunk 5 above)
Comment (GAUTAMRAJU15) [reply src/components/Sidebar/Sidebar.js]: done

File: src/components/Sidebar/Sidebar.js [hunk 11]
```diff
@@ -25,54 +26,67 @@ class Sidebar extends Component {
             <h5></h5>
             <p className="text-muted"></p>
           </div>
-          <div className="topbar-left mt-3 ml-2 pt-2 pl-2" style={{width:'200px'}}>
+          <div
+            className="topbar-left mt-3 ml-2 pt-2 pl-2"
+            style={{ width: '200px' }}
```
Comment (ShankyRana) [inline src/components/Sidebar/Sidebar.js]: dont use inline style

File: src/components/Sidebar/Sidebar.js [hunk 12]
```diff
@@ -25,54 +26,67 @@ class Sidebar extends Component {
...
-
           <div id="sidebar-menu">
             <div className="button-list">
-              <Link to="/new"><button type="button" className="btn  btn-pink waves-effect  ml-4 p-2  pt-0 pb-0 mb-4 w-lg "><i className="fi-plus " style={{fontSize:'15px'}}></i>&nbsp; <span className="h5">New</span> </button></Link>
+              <Link to="/new">
+                <button
+                  type="button"
+                  className="btn btn-pink waves-effect  ml-4 p-2  pt-0 pb-0 mb-4 w-lg "
+                >
+                  <i className="fi-plus " style={{ fontSize: '15px' }} />&nbsp;{' '}
```
Comment (ShankyRana) [inline src/components/Sidebar/Sidebar.js]: remove inline style

File: src/components/Sidebar/Sidebar.js [hunk 13]
```diff
@@ -25,54 +26,67 @@ class Sidebar extends Component {
...
+                        </Link>
+                      }
+                    </li>
+                  );
+                return null;
+              })}
               <li className="menu-title">More</li>
               <li>
-                <a href ="#dashboard"><i className="fi-location-2"></i> <span> Beta Features </span> <span className="menu-arrow"></span></a>
+                <a href="#dashboard">
```
Comment (ShankyRana) [inline src/components/Sidebar/Sidebar.js]: remove # from the links

File: src/components/Sidebar/Sidebar.js [hunk 14]
```diff
@@ -25,54 +26,67 @@ class Sidebar extends Component {
...
+                  <i className="fi-location-2" /> <span> Beta Features </span>{' '}
+                  <span className="menu-arrow" />
+                </a>
                 <ul className="nav-second-level" aria-expanded="false">
-                  <li><a href="#dashboard">Live Stream</a></li>
+                  <li>
+                    <a href="#dashboard">Live Stream</a>
```
Comment (ShankyRana) [inline src/components/Sidebar/Sidebar.js]: remove #

File: src/components/Profile/Profile.js [hunk 15]
```diff
@@ -125,23 +128,52 @@ class Profile extends Component {
...
+    });
+  }
 
 
   render() {
     const isDisabled = this.state.profileState === 'edit'
       ? true
       : false;
     const profile = this.state;
+    console.log(this.state.selectedCountry);
```
Comment (ShankyRana) [inline src/components/Profile/Profile.js]: remove console.

File: src/components/Profile/Profile.js [hunk 16]
```diff
@@ -125,23 +128,52 @@ class Profile extends Component {
...
+      <option key={i}  value={data.CountryName}>
+        {data.CountryName}
       </option>
     ));
   }
 
 
+  getStateRows = () => {
+    let countryList = this.state.countryList.filter(country => country.CountryName === this.state.selectedCountry);
+    return countryList.map(country => {
```
Comment (ShankyRana) [inline src/components/Profile/Profile.js]: try to minimize the iterations.

File: src/components/Profile/Profile.js [hunk 17]
```diff
@@ -125,23 +128,52 @@ class Profile extends Component {
...
+        return <option key={i}  value={state.StateName}>
+          {state.StateName}
+        </option>;
+      });
+    });
+  }
+
 
+  getCityRows = () => {
+    let countryList = this.state.countryList.filter((states) => states.States.StateName === this.state.selectedState);
```
Comment (ShankyRana) [inline src/components/Profile/Profile.js]: minimize the iterations.

File: src/components/Dashboard/Dashboard.js [hunk 18]
```diff
@@ -3,12 +3,12 @@ import { connect } from 'react-redux';
 import { browserHistory } from 'react-router';
 import { Row, Col } from 'react-bootstrap';
 import moment from 'moment';
-
-import { Scrollbars } from 'react-custom-scrollbars';
+import Circle from 'react-circle';
+//import { Scrollbars } from 'react-custom-scrollbars';
```
Comment (ShankyRana) [inline src/components/Dashboard/Dashboard.js]: remove comments

File: src/components/Dashboard/Dashboard.js [hunk 19]
```diff
@@ -137,7 +137,163 @@ class Dashboard extends Component {
...
+            </Col>
+          </Row>
+
+
+
+
+
+
+
+          {/* <Row>
```
Comment (ShankyRana) [inline src/components/Dashboard/Dashboard.js]: remove comments

File: src/components/NotificationSettings/NotificationConfigure/NotificationConfigure.scss (hunk 9 above)
Comment (GAUTAMRAJU15) [reply src/components/NotificationSettings/NotificationConfigure/NotificationConfigure.scss]: done

File: src/components/Sidebar/Sidebar.js (hunk 3 above)
Comment (GAUTAMRAJU15) [reply src/components/Sidebar/Sidebar.js]: this will be done later as of now , pls merge this , dev sir will be workin

**Pull request closed without merging**